examples/camera-app/linux/src/clusters/webrtc-provider/webrtc-provider-manager.cpp

```diff
@@ -440,6 +445,92 @@ WebRTCProviderManager::ValidateStreamUsage(StreamUsageEnum streamUsage,
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR WebRTCProviderManager::ValidateVideoStreamID(uint16_t videoStreamId)
+{
+    if (mCameraAVStreamMgmtServer == nullptr)
+    {
+        ChipLogError(Camera, "AV Stream Management Server not available for video stream validation");
+        return CHIP_ERROR_INCORRECT_STATE;
+    }
+
+    const std::vector<VideoStreamStruct> & allocatedVideoStreams = mCameraAVStreamMgmtServer->GetAllocatedVideoStreams();
+
+    // Check if the videoStreamId exists in allocated streams
+    for (const auto & stream : allocatedVideoStreams)
+    {
+        if (stream.videoStreamID == videoStreamId)
+        {
+            ChipLogProgress(Camera, "Video stream ID %u is valid and allocated", videoStreamId);
+            return CHIP_NO_ERROR;
+        }
+    }
+
+    ChipLogError(Camera, "Video stream ID %u not found in allocated video streams", videoStreamId);
+    return CHIP_ERROR_INVALID_ARGUMENT;
+}
+
+CHIP_ERROR WebRTCProviderManager::ValidateAudioStreamID(uint16_t audioStreamId)
+{
+    if (mCameraAVStreamMgmtServer == nullptr)
+    {
+        ChipLogError(Camera, "AV Stream Management Server not available for audio stream validation");
+        return CHIP_ERROR_INCORRECT_STATE;
+    }
+
+    const std::vector<AudioStreamStruct> & allocatedAudioStreams = mCameraAVStreamMgmtServer->GetAllocatedAudioStreams();
+
+    // Check if the audioStreamId exists in allocated streams
+    for (const auto & stream : allocatedAudioStreams)
+    {
+        if (stream.audioStreamID == audioStreamId)
+        {
+            ChipLogProgress(Camera, "Audio stream ID %u is valid and allocated", audioStreamId);
+            return CHIP_NO_ERROR;
+        }
+    }
+
+    ChipLogError(Camera, "Audio stream ID %u not found in allocated audio streams", audioStreamId);
+    return CHIP_ERROR_INVALID_ARGUMENT;
+}
+
+CHIP_ERROR WebRTCProviderManager::IsPrivacyModeActive(bool & isActive)
+{
+    if (mCameraAVStreamMgmtServer == nullptr)
+    {
+        ChipLogError(Camera, "AV Stream Management Server not available");
+        return CHIP_ERROR_INCORRECT_STATE;
+    }
+
+    // Check privacy mode attributes
```

Comment:
Would recommend calling the SDK CameraAVStreamMgmtServer methods from inside a public AVSM Delegate that this function can invoke.

Reply:
Done